tests/FSharp.Compiler.Service.Tests/AssemblyContentProviderTests.fs

```diff
@@ -63,7 +63,7 @@ let private getSymbolMap (getSymbolProperty: AssemblySymbol -> 'a) (source: stri
     |> List.map (fun s -> getCleanedFullName s, getSymbolProperty s)
     |> Map.ofList
 
-[<Fact>]
+[<FactForDESKTOP>]
```

Comment:
Why?

Reply:
They are failing there, could reproduce locally on WSL. Agreed this should be fixed, I created the ticket for these and other similar tests we have: #17336

Couldn't come with a super quick fix, it might be some actual problem with code itself and not tests. We are getting wrong symbol names, like:
```
Expected
["Test"; "Test.MyType"; "Test.MyType"; "Test.MyType.func123"]
but was
["C:\test"; "C:\test.MyType"; "C:\test.MyType"; "C:\test.MyType.func123"]
```

Either way, this is orthogonal to script fixes here, they have only uncovered the problem, because the shell script was not actually running this before.

Reply:
Why did they start failing all of the sudden with no changes other than script ones? Why don't they fail in `main` or release CI?

Reply:
Because they were being executed via that `TestUsingXUnit` function that didn't exist. If you run them from main on WSL via plain `./build.sh -c Release --testCoreClr`, they fail.

Now, since the scripts are fixed, they are just executed.

Reply:
We are actually very lucky these are the only ones.

Reply:
These failures are not good, it means we're (or bcl) likely broke some of the public surface scenarios.

Reply:
Yes, that's not good at all.